### PR TITLE
Remove diagnostics

### PR DESCRIPTION
**Pull request by tomyyyD**
Moving diagnostics stuff into the middleware to simplify and optimize for memory

### CODE REVIEW

File: flight/tasks/imu.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ async def main_task(self):
             # SATELLITE.IMU.disable()
 
             log_data = {
-                "time": time.time(),
+                "time": int(time.time()),
```
Comment (Ibrassow) [inline flight/tasks/imu.py:51]: CP's time.time() returns already an int (see docs) and is redundant. Adapting the behavior of time for the emulator should be done externally (mock/etc)

File: flight/tasks/monitor.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ async def main_task(self):
 
         # Read data for monitoring
         readings = {
-            "time": time.time(),
+            "time": int(time.time()),
```
Comment (Ibrassow) [inline flight/tasks/monitor.py:50]: Same as above

File: flight/tasks/sun.py [hunk 3]
```diff
@@ -38,7 +38,7 @@ async def main_task(self):
             )
 
             readings = {
-                "time": time.time(),
+                "time": int(time.time()),
```
Comment (Ibrassow) [inline flight/tasks/sun.py:41]: same

File: flight/apps/comms/radio_helpers.py [hunk 4]
```diff
@@ -340,6 +341,9 @@ def transmit_message(self):
                 # Transmit SAT heartbeat or ACK
                 tx_message = construct_message(self.gs_req_message_ID)
 
+            # TODO: remove
+            msg = hexlify(bytes([0xFF, 0xFF, 0x00, 0x00]) + tx_message)
+            print(f"[100][SERIAL OUTPUT]:{msg}")
```
Comment (Ibrassow) [inline flight/apps/comms/radio_helpers.py:346]: Really not a fan. Still necessary?

Comment (tomyyyD) [reply flight/apps/comms/radio_helpers.py:346]: yes

**Review by Ibrassow — APPROVED** [review]

**Pull request merged**